packages/react-server-cli/src/index.js

```diff
@@ -1,3 +1,7 @@
-import startServer from "./startServer"
+require.extensions['.css'] = function(module, filename) {
+	return module._compile("", filename);
+};
 
-export {startServer as start}
```

Comment:
I wanted to put the require extension before the call to `require` to be 100% sure that no CSS gets loaded. There's no way to do that with `import` 😢